views/question.pug

```diff
@@ -20,18 +20,33 @@ block content
   .container
     .row
       .col-md-8
-        form.form-question#answerform
-          h2.qtext #{question.title}
-          h5.qtext !{question.text}
-          label.sr-only(for='answer') Answer
-          if answered
-            input.form-control#answer(name='answer' placeholder='Enter your answer here'
-                                      required disabled)
-            button(class='btn btn-lg btn-primary btn-block' type='submit' disabled) Submit
-          else
-            input.form-control#answer(name='answer' placeholder='Enter your answer here'
-                                      required autofocus)
-            button(class='btn btn-lg btn-primary btn-block' type='submit') Submit
+
+        h2.qtext #{question.title}
+        h5.qtext !{question.text}
+        case question.type
+          when 0
```

Comment:
what is 0 and what is 1, documentation please

Reply:
documentation is in server\questions.js line 28,29